pages/settings/settings.js

```diff
@@ -100,7 +100,8 @@ for (var contentType in contentTypes) {
   (function (contentType) {
     settings.get('filtering', function (value) {
       // create the settings section for blocking each content type
-
+      if ( value == undefined ) 
```

Comment:
Value is already checked below, so I don't think this is necessary.

Reply:
The looping function recreates elements everytime a value changes

Reply:
I'm not sure I understand what you mean, can you clarify?

This function is creating a checkbox for each content type, and if the value is true, it sets the checkbox to checked based on that. `Value`s existence is already checked on line 114, so an undefined value shouldn't cause an issue, and we still want the rest of the function to run (so that the checkboxes still appear).